Week-3/Session9 Basics JS/strongPasword.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Password</title>
+    <script>
+      function isStrongPassword(password) {
+        let str = Array.from(password);
+        let uppercase;
+        let check;
+        
+        for (let i = 0; i < str.length; i++) {
+          if (str[i].valueOf().toUpperCase() === str[i].valueOf()){
+           uppercase = true;
+            break;
+          } else {
+            uppercase = false;
+          }
+        }
```

Comment:
refactor to use password.some()

Reply:
str=["e","E","F","f"];
let uppercase;
function isUpperCase() {
     for (let item of str) {
      if (item.valueOf().toUpperCase() === item.valueOf())
         return true;
       return false;
     }
  }
   str.some(isUpperCase) ? uppercase = true: uppercase = false;
 console.log(uppercase);


I tried using some, but its returning false for every iteration every when it is true